build issue on OSX 10.10.1

After running the "grunt" command i got this error:


Running "webpack-dev-server:start" (webpack-dev-server) task
webpack-dev-server on port 8089  
Version: webpack 1.4.15     
                                Asset     Size  Chunks             Chunk Names
 7ad17c6085dee9a33787bac28fb23d46.eot    20335          [emitted]  
 e49d52e74b7689a0727def99da31f3eb.ttf    41280          [emitted]  
 ff423a4251cf2986555523dfe315c42b.svg    62926          [emitted]  
68ed1dac06bf0409c18ae7bc62889170.woff    23320          [emitted]  
692b2ac094cb0b2679dadd8cba568087.woff    24680          [emitted]  
                            bundle.js  3258457       0  [emitted]  main

ERROR in ./app/components/Tools.jsx
Module not found: Error: Cannot resolve 'file' or 'directory' ./SubWithdraw in /Users/aurel/etherex/frontend/app/components
 @ ./app/components/Tools.jsx 15:18-42
webpack: bundle is now VALID.

Now it looks like this 

Iancus-MacBook-Pro:frontend aurel$ grunt
Running "webpack-dev-server:start" (webpack-dev-server) task
webpack-dev-server on port 8089  
Version: webpack 1.4.15     
                                Asset     Size  Chunks             Chunk Names
68ed1dac06bf0409c18ae7bc62889170.woff    23320          [emitted]  
692b2ac094cb0b2679dadd8cba568087.woff    24680          [emitted]  
 7ad17c6085dee9a33787bac28fb23d46.eot    20335          [emitted]  
 e49d52e74b7689a0727def99da31f3eb.ttf    41280          [emitted]  
 ff423a4251cf2986555523dfe315c42b.svg    62926          [emitted]  
                            bundle.js  3350362       0  [emitted]  main

ERROR in ./~/ethereum.js/lib/contract.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./web3 in /Users/aurel/etherex/frontend/node_modules/ethereum.js/lib
 @ ./~/ethereum.js/lib/contract.js 25:15-32

ERROR in ./~/ethereum.js/lib/abi.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./web3 in /Users/aurel/etherex/frontend/node_modules/ethereum.js/lib
 @ ./~/ethereum.js/lib/abi.js 26:15-32

ERROR in ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js
Module not found: Error: Cannot resolve module 'child_process' in /Users/aurel/etherex/frontend/node_modules/ethereum.js/node_modules/xmlhttprequest/lib
 @ ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js 15:12-36

ERROR in ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js
Module not found: Error: Cannot resolve module 'fs' in /Users/aurel/etherex/frontend/node_modules/ethereum.js/node_modules/xmlhttprequest/lib
 @ ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js 16:9-22


Thanks for the reply @caktux !

Still getting XMLHttpRequest.js errors:
```
ERROR in ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js
Module not found: Error: Cannot resolve module 'child_process' in /.../etherex/frontend/node_modules/ethereum.js/node_modules/xmlhttprequest/lib
 @ ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js 15:12-36

ERROR in ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js
Module not found: Error: Cannot resolve module 'fs' in /.../etherex/frontend/node_modules/ethereum.js/node_modules/xmlhttprequest/lib
 @ ./~/ethereum.js/~/xmlhttprequest/lib/XMLHttpRequest.js 16:9-22
```

--- a/frontend/webpack.config.js
+++ b/frontend/webpack.config.js
@@ -25,5 +25,8 @@ module.exports = {
       { test: /\.jpg/, loader: 'url-loader?limit=10000&minetype=image/jpg' },
       { test: /\.png/, loader: 'url-loader?limit=10000&minetype=image/png' }
     ]
-  }
+  },
+  externals:[{
+    xmlhttprequest: '{XMLHttpRequest:XMLHttpRequest}'
+  }]
 };

